src/pudl/metadata/fields.py

```diff
@@ -801,6 +801,11 @@
     "highest_distribution_voltage_kv": {"type": "number", "unit": "kV"},
     "home_area_network": {"type": "integer"},
     "inactive_accounts_included": {"type": "boolean"},
+    "income_type": {
+        "type": "string",
+        "description": "Type of income reported in income_statement_ferc1 table.",
+    },
+    "income": {"type": "number", "description": "Dollars of income."},
```

Comment:
halp! $s of income... doesn't feel like a real description but idk what else to say. it's $s of income for specific `income_type` and `utility_type`?